haunt-roll-fail/arcs/game-leaders.scala

```diff
@@ -192,7 +192,8 @@ object LeadersExpansion extends Expansion {
                 if (f.can(Cryptic))
                     f.outraged ++= $(Material, Fuel)
 
-
+                if (f.lores.has(AncientHoldings))
+                    f.keys = $(3, 1, 4, 1, 2, 1, 3)
```

Comment:
Maybe it could be the very first slot, not "in the middle"?

Reply:
Doing so interferes a bit with leader setups, where the resources should go to the first 2 slots.